src/librustc/traits/drop.rs

```diff
@@ -0,0 +1,202 @@
+//! Miscellaneous type-system utilities that are too small to deserve their own modules.
```

Comment:
The module name is a bit misleading (since it's not just about `Drop`); I'd name it `misc.rs` or something.